models/plan.py

```diff
@@ -111,3 +108,26 @@ def close_camps(self, *camps: Camp) -> None:
             raise self.MissingCampsError()
         else:
             self.save()
+
+    def close(self, __is_closed=None):
+        """
+        Set __is_closed flag to be True if plan is closed.
+        """
+        self.__is_closed = True
+        self.__close_date = datetime.today().date()
+        self.save()
+
+    @property
+    def close_date(self) -> date:
+        """
+        Get the read only close date of the plan.
+        """
+        if self.__is_closed:
+            return self.__close_date
```

Comment:
Since you are not returning anything if `self.__is_closed == false`, it is the same without the if condition check. 

to be more clear, you can specify that None will be returned if plan is not closed in the docstring, and change the return type hint to `Union[date, None]`

Reply:
Thank you, I've changed the doc string and return type hint according to what you said!